Classes/Plugins/RelatedListTool/RelatedListTool.php

```diff
@@ -154,19 +172,22 @@ public function getRelatedItems()
             $title = (string) $relatedItemXmlElement->xpath('mods:titleInfo/mods:title')[0];
             $docId = (string) $relatedItemXmlElement->xpath('mods:identifier[@type="' . $type . '"]')[0];
             $order = (string) $relatedItemXmlElement->xpath('mods:part/@order')[0];
+            $volume = (string) $relatedItemXmlElement->xpath('mods:part[@type="volume"]/mods:detail/mods:number')[0];
 
             $element = array();
             $element['type'] = $type;
             $element['title'] = $title;
             $element['docId'] = $docId;
-            $element['order'] = (is_numeric($order)) ? (int) $order : 0;
+            $element['order'] = (is_numeric($order)) ? (int) $order : PHP_INT_MAX;
+            $element['volume'] = (!empty($volume)) ? $volume : null;
 
             $relatedItems[$index] = $element;
         }
 
-        usort($relatedItems, array('RelatedListTool', 'compareBySortOrder'));
+        usort($relatedItems, array('EWW\Dpf\Plugins\RelatedListTool\RelatedListTool', 'compareBySortOrder'));
```

Comment:
This works maybe, but you should either double escape the namespace path inside a string or use the ::class constant.

usort($relatedItems, array('EWW\\Dpf\\Plugins\\RelatedListTool\\RelatedListTool', 'compareBySortOrder'));
or (prefered)
usort($relatedItems, array(EWW\Dpf\Plugins\RelatedListTool\RelatedListTool::class, 'compareBySortOrder'));

As we require PHP 5.6 as minimum. This would be safe.